File: Models/DbContext.cs
using KhumaloCraft.Models;
using System.Collections.Generic;
using Microsoft.EntityFrameworkCore;

namespace KhumaloCraft.Models;

public class MyDbContext : DbContext
{
    public MyDbContext(DbContextOptions<MyDbContext> options)
    : base(options) { }

    public DbSet<User> Users { get; set; }
    public DbSet<Product> Products { get; set; }
    public DbSet<ContactMessage> ContactMessages { get; set; }
}

File: KhumaloCraft/Data/DbContext.fs
namespace KhumaloCraft.Data

open Microsoft.EntityFrameworkCore
open KhumaloCraft.Entities

type MyDbContext (options: DbContextOptions<MyDbContext>) =
  inherit DbContext(options)

  // Define DbSet properties for each entity
  member val Users: DbSet<User> = this.Set<User>()
  member val Products: DbSet<Product> = this.Set<Product>()
  member val ContactMessages: DbSet<ContactMessage> = this.Set<ContactMessage>()
